### ISSUE TITLE
keyboard shortcuts not immediately active when opening an email in new tab

### ISSUE THREAD

**Issue by ewtoombs** [protz/thunderbird-conversations#1147]
I hide the main message view and open messages individually, one per tab, with the keyboard by selecting the message and hitting enter. This keyboard navigation saves me a lot of time. Conversations puts a wrench in this though. After I open an email with enter, paging with page up and down doesn't work immediately until after I click on the email text. Very annoying. So, could you change the default keyboard focus to the message body when an email is opened with enter?

**Comment by ewtoombs** [comment]
workaround: If I hit tab once after opening the email with enter, the focus is on the right pane. That isn't necessary or desirable though.

**Comment by ewtoombs** [comment]
The problem is worse than I thought. ctrl-R for reply doesn't work either until after you hit tab to select the message pane. On top of that, ctrl-R doesn't use the inline reply interface. Looks like keyboard navigation generally needs some polish.

**Comment by protz** [comment]
Looks like there's no keyboard shortcut to jump to quick reply. Would
gladly take a pull request that adds one!

**Comment by Standard8** [comment]
We've just removed the overridden keyboard shortcuts from the latest master, so this should be improved in the next major version (probably 3.x) of Conversations.
